File: C# II/03. Methods/Methods-Homework/03. EnglishDigit/EnglishDigit.cs
using System;

    class EnglishDigit
    {
        /*
          3. English digit

         Write a method that returns the last digit of given integer 
         as an English word.

        */

        static string EnglishDigitMethod(int number)
        {
            Console.Write("\nLast digit as word: ");
            string[] numbers = {
              "zero", "one", "two", "three", "four", "five",
              "six", "seven", "eight", "nine"
            };
            return numbers[number % 10];
        }

        static void Main(string[] args)
        {
            Console.Write("Enter an integer: ");
            Console.WriteLine(EnglishDigitMethod(Convert.ToInt32(Console.ReadLine())));
            Console.WriteLine();
        }
    }

File: JavaScript/08. Functions/Functions-Homework/Functions-Homework/01. EnglishDigit/EnglishDigit.js
/*
 * Problem 1. English digit

Write a function that returns the last digit of given integer as an English word.
 */

var arr = [512, 1024, 12309],
    i,
    len;

for (i = 0, len = arr.length; i < len; i += 1) {
    EnglishDigit(arr[i]);
}

function EnglishDigit(number) {
    console.log("Number: " + number + " last digit: ");
    number = number % 10;
    switch (number) {
        case 0:
            console.log("zero");
            break;
        case 1:
            console.log("one");
            break;
        case 2:
            console.log("two");
            break;
        case 3:
            console.log("three");
            break;
        case 4:
            console.log("four");
            break;
        case 5:
            console.log("five");
            break;
        case 6:
            console.log("six");
            break;
        case 7:
            console.log("seven");
            break;
        case 8:
            console.log("eight");
            break;
        case 9:
            console.log("nine");
            break;
        default:
            console.log("is not a number");
            break;
    }
}